packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -762,6 +811,14 @@ export default {
                     this.shouldShowSpinner = true;
                 }
             }, 1000);
+        },
+
+        handleErrorDialogClick () {
+            if (this.nonFulfillableError.shouldRedirectToMenu) {
+                window.location.assign(this.restaurantMenuPageUrl);
```

Comment:
Maybe a better approach would be to use an anchor with a `href` value set rather than a button to link to the restaurant page? 

You can still hook click events onto the anchor a then call `e.preventDefault()` if the user shouldn't be redirected.